healpix_alchemy/tests/test_regions.py

```diff
@@ -101,3 +102,23 @@ def test_fields(request, session, engine):
 
     fields = session.query(Field).all()
     assert len(fields) == 18
+
+
+def test_tile(engine):
+    """Generate an example Tile and confirm index calculation."""
+    Base.metadata.create_all(engine)
+
+    LEVEL = MOC.HPY_MAX_NORDER
+
+    value = level_ipix_to_uniq(3, 12)
+
+    # @uniq.setter
```

Comment:
The unit test should check the methods themselves, not a copies of them.

Reply:
But I am checking the Tile output right based on the calculation? Maybe I don't understand what you mean? Or do you just want the values of nested_lo and nested_hi hardcoded?

Reply:
This unit test does show that your method is correct, but I would like you to unit test the actual getter and setter methods, not simply copy-paste their bodies into the unit test.